ocs_ci/ocs/resources/pod.py

```diff
@@ -923,6 +923,28 @@ def get_noobaa_core_pod():
     return noobaa_core_pod
 
 
+def get_noobaa_db_pod():
+    """
+    Get noobaa db pod obj
+
+    Returns:
+        Pod object: Noobaa db pod object
+
+    """
+    if version.get_semantic_ocs_version_from_config() > version.VERSION_4_6:
+        nb_db = get_pods_having_label(
+            label=constants.NOOBAA_DB_LABEL_47_AND_ABOVE,
+            namespace=config.ENV_DATA["cluster_namespace"],
+        )
+    else:
+        nb_db = get_pods_having_label(
+            label=constants.NOOBAA_DB_LABEL_46_AND_UNDER,
+            namespace=config.ENV_DATA["cluster_namespace"],
+        )
```

Comment:
We can omit the check since it's always true:
```suggestion
        nb_db = get_pods_having_label(
            label=constants.NOOBAA_DB_LABEL_47_AND_ABOVE,
            namespace=config.ENV_DATA["cluster_namespace"],
        )
```